### ISSUE TITLE
Items are spawning at 0,0 again.

### ISSUE THREAD

**Issue by TimrodDX** [space-wizards/space-station-14#3793]
![image](https://user-images.githubusercontent.com/7907891/113444775-8937b500-93c2-11eb-83d5-8ab5fbcba790.png)

![image](https://user-images.githubusercontent.com/7907891/113445049-1549dc80-93c3-11eb-87f0-501cd14e82cd.png)

I don't quite know what makes them spawn at 0,0 but it's just the sprites. More stuff gets added to the pile as the round goes on, and I'm not sure what causes it - I've seen wearables, floor tiles, guns, stun batons, and other stuff get moved there.

**Comment by mirrorcult** [comment]
almost certainly PVS issue

**Comment by Mith-randalf** [comment]
Not sure if helpful, but I've also seen interaction text for 'items' floating out of the pile; mostly monkey cubes getting combined with water.

**Comment by 20kdc** [comment]
This is getting even worse than it seems.
To summarize, while I am not *100%* sure that this is related, it seems it:
The client is receiving entities where the parent isn't being received. The result is that an exception occurs in TransformComponent.cs:687 `var newParent = Owner.EntityManager.GetEntity(newParentId);` because `newParentId` is an entity the client doesn't have.
*Then prediction kicks in.*
Prediction sends this into a death spiral, many many exceptions being caused by applying states, which slows the game down requiring more prediction to correct for them.

**Comment by 20kdc** [comment]
I suspect it's related to containers that are out of PVS somehow. I will attempt to get a more conclusive result now.

**Comment by 20kdc** [comment]
Not sure it's containers, but convincing someone to run around the station with a nar-sie plushie did cause the sprite to appear at 0,0 when the plushie *returned*.
Notably, the original plushie is separate to this.
(As far as I am aware no other instances of the plushie exist but cannot be certain.)
![image](https://user-images.githubusercontent.com/22304167/118883362-112f3b00-b8ed-11eb-9336-ccaf33a664c4.png)
(With thanks to Timmy Two-Toes.)

**Comment by PaulRitter** [comment]
@Zumorica you fixed this, right?

**Comment by Zumorica** [comment]
> @Zumorica you fixed this, right?

Not quite. I fixed a bug that Sloth introduced where deleting things would make their sprite appear on 0,0.
HOWEVER, Sloth's rendering tree cleanup *might* have fixed this, as I couldn't replicate this issue anymore with his cleanup.
Also, I did encounter a bug where a random sprite of a box would appear somewhere random and not disappear. No idea if it's the same bug as this, or something else completely.

**Comment by Ygg01** [comment]
Closed because no replication.
